processor/src/main/java/org/mapstruct/ap/internal/model/ValueMappingMethod.java

```diff
@@ -61,7 +67,18 @@ public Builder valueMappings(List<ValueMappingOptions> valueMappings) {
             return this;
         }
 
-        public ValueMappingMethod build( ) {
+        public Builder enumMapping(EnumMappingOptions enumMapping) {
+            this.enumMapping = enumMapping;
+            return this;
+        }
+
+        public ValueMappingMethod build() {
+
+            if ( !enumMapping.isValid() ) {
```

Comment:
Null check required or could it never be null when `build` is called? The `initializeEnumTransformationStrategy` that will be called after this if-clause includes a `null` check

Reply:
Good spot. I'll need to verify this again. In theory it can never be `null`, since we always create it (like the `BeanMappingOptions`). All tests are green as well, so the if condition is never hit it seems.